apps/drec-ui/src/AppContainer/AppContainer.effects.tsx

```diff
@@ -0,0 +1,69 @@
+import {
+    getAccountMenu,
+    getAdminMenu,
+    getOrganizationMenu,
+    getDeviceMenu,
+    useTopbarButtonList,
+    isRole
+} from '../shared';
```

Comment:
I think it is not the best idea to store those functions in `shared` folder. 
Each part of app should have a dedicated folder (User, Organization, etc.) and general things should be in `utils`.

Reply:
Yeah, I was thinking that adding everything in shared folder is not a good idea.